src/coreclr/jit/lower.cpp

```diff
@@ -9686,14 +9689,14 @@ void Lowering::ContainCheckRet(GenTreeUnOp* ret)
 {
     assert(ret->OperIs(GT_RETURN, GT_SWIFT_ERROR_RET));
 
-#if !defined(TARGET_64BIT)
+#if !defined(TARGET_64BIT) && !defined(TARGET_WASM)
```

Comment:
Nit - this seems redundant: https://github.com/dotnet/runtime/blob/ed58e5fd2d5bce794c1d5acafa9f268151fefd47/eng/native/configurecompiler.cmake#L522-L527